sse2neon.h

```diff
@@ -1692,20 +1692,16 @@ FORCE_INLINE __m128i _mm_srai_epi16(__m128i a, int imm)
 // Shifts the 4 signed or unsigned 32-bit integers in a left by count bits while
 // shifting in zeros. :
 // https://msdn.microsoft.com/en-us/library/z2k3bbtb%28v=vs.90%29.aspx
-// FORCE_INLINE __m128i _mm_slli_epi32(__m128i a, __constrange(0,255) int imm)
```

Comment:
Let's move `__constrange(0,255)` back.  Someday, we can introduce something like [SIMDE_REQUIRE_CONSTANT_RANGE](https://github.com/simd-everywhere/simde/blob/master/simde/simde-common.h#L186) to validate imm8 at build time.